### ISSUE TITLE
Cannot read property 'isCancellationRequested' of undefined

### ISSUE THREAD

**Issue by linonetwo** [microsoft/pyright#576]
**Describe the bug**
It always errors:

```
[Error - 3:57:57 PM] Request textDocument/references failed.
  Message: Request textDocument/references failed with message: Cannot read property 'isCancellationRequested' of undefined
  Code: -32603 
```

**To Reproduce**
I don't know how... it just occurs one day this week. It happens in every python file.

**Expected behavior**
Maybe check `xxx && xxx.isCancellationRequested`?

**Screenshots or Code**
<img width="1440" alt="Screen Shot 2020-03-21 at 3 59 45 PM" src="https://user-images.githubusercontent.com/3746270/77222303-fe0bc880-6b8c-11ea-8dae-8902fbe568ce.png">

**VS Code extension or command-line**
VS Code extension 1.1.28

**Comment by erictraut** [comment]
Thanks for reporting the problem. Could you tell me what version of VS Code you have installed?

@heejaechang - looks like this is related to the code you added recently. Any ideas?

**Comment by jakebailey** [comment]
Typo in `referencesProvider.ts`:

```
    constructor(
        parseResults: ParseResults,
        filePath: string,
        referencesResult: ReferencesResult,
        includeDeclaration: boolean,
        evaluator: TypeEvaluator,
        token: CancellationToken
    ) {
        super();
        this._parseResults = parseResults;
        this._filePath = filePath;
        this._referencesResult = referencesResult;
        this._includeDeclaration = includeDeclaration;
        this._evaluator = evaluator;
        this._cancellationToken;  // <-------- here
    }
```

**Comment by erictraut** [comment]
Good catch Jake! I'm surprised that the TS compiler didn't catch this.

**Comment by erictraut** [comment]
Just checked in a fix. This will be addressed in the next published version of Pyright.

**Comment by jakebailey** [comment]
Yeah, especially since it's a clear violation of the contract that something not be undefined if it doesn't say it's undefined...

**Comment by heejaechang** [comment]
Ah, thank u for fixing it. I guess time to add a test over find all references

**Comment by erictraut** [comment]
This is now fixed in version 1.1.29, which I just published.
